snakemake/resources.py

```diff
@@ -20,7 +20,7 @@ class DefaultResources:
     @classmethod
     def decode_arg(cls, arg):
         try:
-            return arg.split("=")
+            return arg.split("=")[0], "=".join(arg.split("=")[1:])
```

Comment:
Wouldn't this be the more idiomatic way?
```suggestion
            return arg.split("=", maxsplit=1)
```

Reply:
Yes, thanks for the suggestion! I wasn't aware of the maxsplit argument.